### PR TITLE
New method to test existing FQN value

### PR DESCRIPTION
**Pull request by jonaromero86**
This method is comparing only 1 specific FQN Term.
Before the assert, I can compare manually the Term values, to use other type of asserts like assertTrue or fail.
Let me know, thanks.

### CODE REVIEW

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java [hunk 1]
```diff
@@ -65,4 +85,43 @@ public void testSemantic() {
...
+        String expectedTermFqn = "Erythema gyratum repens (disorder)";
+        String actualTermFqn = "";
+        String actualUUID = "ba8a3539-31fa-307a-895c-37401f0aea78"; //SNOMED CT integer ID: 77300003
+
+        UUID diseaseId = UUID.fromString(actualUUID);
+        Entity<EntityVersion> cldEntity = EntityService.get().getEntityFast(diseaseId);
+
+        long timestamp = dateStringToEpochMillis("20250931");
+        StampPositionRecord stampPosition = StampPositionRecord.make(timestamp, TinkarTerm.DEVELOPMENT_PATH.nid());
+        StampCalculator stampCalc = StampCoordinateRecord.make(StateSet.ACTIVE, stampPosition).stampCalculator();
```
Comment (stenkarl) [inline snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java]: I know I had said to use Ryan's gist as a model, and it's good you did, but for consistency with Patrick's test, can you change the stampCalc here to be "Calculators.Stamp.DevelopmentLatestActiveOnly()" like line 72 above? Then you can remove the stampPosition and timstamp variables, and dateStringToEpochMillis helper method.

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java [hunk 2]
```diff
@@ -65,4 +85,43 @@ public void testSemantic() {
...
+        EntityService.get().forEachSemanticForComponentOfPattern(cldEntity.nid(), TinkarTerm.DESCRIPTION_PATTERN.nid(), (descriptionSemantic) -> {
+            Latest<SemanticEntityVersion> latestDescriptionSemantic = stampCalc.latest(descriptionSemantic);
+            Component descriptionType = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_TYPE, latestDescriptionSemantic.get());
+            if (PublicId.equals(descriptionType.publicId(), TinkarTerm.FULLY_QUALIFIED_NAME_DESCRIPTION_TYPE)) {
+                fqnVersion.set(latestDescriptionSemantic.get());
+            }
+        });
+
+        actualTermFqn = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.TEXT_FOR_DESCRIPTION, fqnVersion.get());
+//        System.out.println("actualTermFqn: " + actualTermFqn);
```
Comment (stenkarl) [inline snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java]: Can you remove this commented out code?

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java [hunk 3]
```diff
@@ -65,4 +85,43 @@ public void testSemantic() {
...
+     */
+    @Test
+    public void testFQN() {
+        // Given
+        String expectedTermFqn = "Erythema gyratum repens (disorder)";
+        String actualTermFqn = "";
+        String actualUUID = "ba8a3539-31fa-307a-895c-37401f0aea78"; //SNOMED CT integer ID: 77300003
+
+        UUID diseaseId = UUID.fromString(actualUUID);
+        Entity<EntityVersion> cldEntity = EntityService.get().getEntityFast(diseaseId);
```
Comment (stenkarl) [inline snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java]: As a small nit, the "cld" in cldEntity stands of "chronic lung disease", but the test no longer refers to that term, so you can just rename this to "entity".

File: snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java (hunk 2 above)
Comment (jonaromero86) [reply snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java]: Sure, will remove the sysout.

**Review by stenkarl — APPROVED** [review]

**Pull request merged**